repository.rdf

```diff
@@ -417,16 +336,6 @@
 
 
 
-    <!-- http://fedora.info/definitions/v4/repository#Configuration -->
-
-    <owl:Class rdf:about="http://fedora.info/definitions/v4/repository#Configuration">
```

Comment:
Hang on-- are we sure about whether or not this would actually break `-transform`? If it would, then we should do some kind of deprecation and maybe even ask whoever is theoretically maintaining `-transform` if they want to migrate the predicate. If not, who cares?